packages/block-library/src/navigation/edit/unsaved-inner-blocks.js

```diff
@@ -157,6 +157,7 @@ export default function UnsavedInnerBlocks( {
 		 */
 		/* eslint-disable react-hooks/exhaustive-deps */
 		[
+			blocks,
```

Comment:
I see that `exhaustive-deps` is disabled for this effect, and there's an extensive comment for the reasoning. Do you still see the warning?

Reply:
Oh I should remove the comment to disable it!

Reply:
Done in [74613e5](https://github.com/WordPress/gutenberg/pull/49291/commits/74613e5f6cd45a8a868460a55cab0f902c66e5b9)